server/views/partials/header.njk

```diff
@@ -1,7 +1,7 @@
 {% from "govuk/components/tag/macro.njk" import govukTag %}
 
-<header class="hmpps-header" role="banner">
-  <div class="hmpps-header__container">
+<header class="govuk-header" role="banner">
```

Comment:
I added these styles to the header so that the header bar gets the blue border at the bottom, as per the design team's designs.